src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -51,6 +51,13 @@ class Prepare(object):
     def __init__(self, argv):
         self.argv = argv
 
+    def get_journal_ptuuid(self, argument):
+        uuid = disk.get_partuuid(argument)
+        if not uuid:
+            terminal.error('blkid could not detect a PARTUUID for device: %s' % argument)
+            raise RuntimeError('unable to use device for a journal')
```

Comment:
Should we mention the 'why' it can't be used as a journal at all here?

Reply:
That might get a bit verbose though, I am inclined to document this rather than adding to the error message. Or where you thinking something succinct?

Reply:
Does the ``terminal.error`` print right there in stdout with the ``RuntimeError``? If so maybe that's enough context. If it was only 'unable to use device for a journal' I wouldn't know why or what to do next.

Reply:
correct, anything `terminal.*`goes to stdout *including* the `RuntimeError`. But still, I need to follow up with docs